### PR TITLE
enable DELAYACCT support

### LINKED ISSUE #4: Delay stats not available
Delay stats like `PERCENT_CPU_DELAY`, `PERCENT_IO_DELAY`, `PERCENT_SWAP_DELAY` are not available as columns in the htop snap.
From the documentation, it seems `CAP_NET_ADMIN` is needed, so it should be just a matter of adding a plug for the `network-control` interface to the snap.

### PR DESCRIPTION
**Pull request by albertodonato**
Support for delay stats display. Fixes #4

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -23,15 +24,20 @@ apps:
       - mount-observe
       - process-control
       - system-observe
+      - network-control
```
Comment (maxiberta) [inline snap/snapcraft.yaml]: Please sort alphabetically.

File: snap/snapcraft.yaml [hunk 2]
```diff
@@ -10,6 +10,7 @@ description: |
 
   Once installed, this snap needs manually connecting to some plugs:
     `sudo snap connect htop:mount-observe`
+    `sudo snap connect htop:network-control`
```
Comment (maxiberta) [inline snap/snapcraft.yaml]: We could add a note somewhere explaining why would `htop` need `network-control` capabilities, e.g. `sudo snap connect htop:network-control  # Optional, DELAYACCT support.` ?

**Review by maxiberta — COMMENTED** [review]
Looks good, thanks! It'll only work for `root` afaict (with `network-control` connected), am I right? Check extra comments below.

File: snap/snapcraft.yaml (hunk 1 above)
Comment (albertodonato) [reply snap/snapcraft.yaml]: done

File: snap/snapcraft.yaml (hunk 2 above)
Comment (albertodonato) [reply snap/snapcraft.yaml]: done

**Comment by albertodonato** [comment]
@maxiberta yes, it requires running `htop` as `root`

**Comment by maxiberta** [comment]
Merged, thanks! This will trigger [Launchpad's auto-build & auto-publish](https://code.launchpad.net/~maxiberta/+snap/htop-beta) to the `beta` channel. Then, you can test the new snap with `sudo snap refresh htop --beta`.

**Comment by maxiberta** [comment]
This is now released in the `stable` channel.

**Pull request merged**